compute/instances/how-to/create-an-instance.mdx

```diff
@@ -6,12 +6,8 @@ content:
   h1: How to create an Instance
   paragraph: This page explains how to create an Instance
 tags: instance create 
-steps:
```

Comment:
I'm not sure about this one - we had to leave it in there as previously the build failed If it was removed - have you tried to make a build?